packages/likec4/src/vite-plugin/virtuals/icons.ts

```diff
@@ -10,8 +10,7 @@ function code(views: ComputedView[]) {
     filter(isString),
     filter(s =>
       isTruthy(s) &&
-      !(s.toLowerCase().startsWith('http:') || s.toLowerCase().startsWith('https:') ||
-        s.toLowerCase().startsWith('file:') || s.toLowerCase().startsWith('.'))
```

Comment:
Dropped support for file: as it's a security risk, and blocked by most browsers.